lib/buyers/views.py

```diff
@@ -0,0 +1,9 @@
+from lib.buyers.models import Buyer
+from lib.buyers.serializers import BuyerSerializer
+from solitude.base import NonDeleteModelViewSet
+
+
+class BuyerViewSet(NonDeleteModelViewSet):
+    queryset = Buyer.objects.all()
```

Comment:
I kinda like using `model = Buyer` when I'm not using a special queryset but that's just me.